channeldb/invoices.go

```diff
@@ -247,6 +247,73 @@ const (
 	HtlcSetBlankModifier
 )
 
+// InvoicesDB defines the interface to for stores to implement for
+// invoice-related functionality.
+//
+// TODO(bitromortac): invert dependency relation between channeldb and invoices
+// and move to invoices.
+type InvoicesDB interface {
```

Comment:
What about moving this to the invoice package itself? Then over time we can add new types there that are defined in that package. Follows the pattern of a package "declaring" what interface needs, with other packages then providing those at instantiation time.